CHANGELOG.md

```diff
@@ -5,15 +5,18 @@
 * PaymentButtons
     * Deprecate `PaymentButtonColor` `.black`, `.silver`, `.blue`, and `.darkBlue`
     * Add `PaymentButtonColor.gold` for `PayPalCreditButton`
+    * Add `custom` case for `PaymentButtonEdges`
+    * Support VoiceOver by adding button accessibility labels
 
 * PayPalWebPayments
   * Add `vault(url:)` method to `PayPalWebCheckoutClient`
   * Add `PayPalVaultResult` type to return vault result
   * Add `PayPalVaultDelegate` to handle results from vault flow
   * Add `PayPalWebCheckoutClientError.paypalVaultResponseError` for missing or invalid response from vaulting
-* PaymentButtons
-  * Add `custom` case for `PaymentButtonEdges`
-  * Support VoiceOver by adding button accessibility labels
+
+* Breaking Changes
```

Comment:
We can remove the `Breaking Changes` label if this is going in as a patch change on `main` (V1)